### PR TITLE
Removed BSF Analytics.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- NA

### Screenshots

- NA

### Types of changes

- NA

### How has this been tested?

- NA

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: .stylelintignore [hunk 1]
```diff
@@ -20,7 +20,6 @@ dist/
 assets/css/blocks/
 assets/css/custom-style-blocks.css
 
-admin/bsf-analytics/
 admin/assets/images/
```
Comment (sandeshjangam) [inline .stylelintignore:23]:
```suggestion
admin/bsf-analytics/
admin/assets/images/
```

File: phpcs.xml.dist [hunk 2]
```diff
@@ -51,7 +51,6 @@
 	<exclude-pattern>*/languages/*</exclude-pattern>
 	<exclude-pattern>*/scripts/*</exclude-pattern>
 	<exclude-pattern>*/admin/assets/*</exclude-pattern>
-	<exclude-pattern>*/admin/bsf-analytics/*</exclude-pattern>
 	<exclude-pattern>/blocks-config/uagb-controls/spectra-icons-v6.php</exclude-pattern>
```
Comment (sandeshjangam) [inline phpcs.xml.dist:54]:
```suggestion
	<exclude-pattern>*/admin/bsf-analytics/*</exclude-pattern>
	<exclude-pattern>/blocks-config/uagb-controls/spectra-icons-v6.php</exclude-pattern>
```

File: classes/class-uagb-admin-helper.php [hunk 3]
```diff
@@ -136,8 +130,6 @@ public static function get_block_options() {
 			$blocks       = UAGB_Helper::$block_list;
 			$saved_blocks = self::get_admin_settings_option( '_uagb_blocks' );
 
-			update_option( 'spectra_saved_blocks_settings', $saved_blocks );
-
```
Comment (sandeshjangam) [inline classes/class-uagb-admin-helper.php:140]: Why this removal

File: classes/class-uagb-admin-helper.php [hunk 4]
```diff
@@ -80,12 +80,6 @@ public static function get_admin_settings_shareable_data() {
 				),
 			);
 
-			$setting_data = get_option( 'spectra_settings_data' );
-
-			if ( ! $setting_data ) {
-				update_option( 'spectra_settings_data', $options );
-			}
-
```
Comment (sandeshjangam) [inline classes/class-uagb-admin-helper.php:88]: Why this removal

File: classes/class-uagb-admin-helper.php (hunk 3 above)
Comment (AbhijitNage123) [reply classes/class-uagb-admin-helper.php:140]: It's called in `class-uagb-inits-blocks.php` on the regenerate_analytics_data function called on `spectra_analytics_complete_action` which is not needed anymore. that is why we removed it.

File: classes/class-uagb-admin-helper.php (hunk 4 above)
Comment (AbhijitNage123) [reply classes/class-uagb-admin-helper.php:88]: It's called in `class-uagb-inits-blocks.php` on the regenerate_analytics_data function called on `spectra_analytics_complete_action` which is not needed anymore. that is why we removed it.

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**